go.mod

```diff
@@ -24,4 +24,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220405142115-a332d4046e57
```

Comment:
Reminder.

Reply:
removed in [bcea49e](https://github.com/codeready-toolchain/toolchain-common/pull/247/commits/bcea49e1558cfb905d26cd6d513f34911f9afebe)